sysutils/kubectx/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ahmetb kubectx v0.6.3
+github.tarball_from archive
```

Comment:
Is this actually necessary, or would the default also work (given a different checksum?)